src/createCycleMiddleware.js

```diff
@@ -1,4 +1,5 @@
 import xs from 'xstream'
+import {adapt} from '@cycle/run/lib/adapt'
```

Comment:
for now i added it in the tests (new commit) to make sure nothing breaks. should be enough to test it there